stats.lua

```diff
@@ -392,6 +402,54 @@ local function append_display_sync(s)
 end
 
 
+local function append_filters(s, prop, prefix)
+    local length = 0
+    local filters = {}
+
+    local speed = mp.get_property_osd("speed")
+    if speed ~= "1.00" then
+        length = length + string.len("speed []") + speed:len()
+        filters[#filters+1] = "speed" .. it(" [" .. speed .. "]")
+    end
+
+    if prop == "vf" and mp.get_property_bool("deinterlace") then
```

Comment:
But yadif is not necessarily the filter that gets inserted. It might be a hwdec specific version that will. And until there is a way to actually query what got inserted, just saying "deinterlace" should be good enough.